pkl-core/src/main/java/org/pkl/core/runtime/VmException.java

```diff
@@ -117,8 +117,8 @@ public String toString() {
   }
 
   @TruffleBoundary
-  public PklException toPklException(StackFrameTransformer transformer) {
-    var renderer = new VmExceptionRenderer(new StackTraceRenderer(transformer));
+  public PklException toPklException(StackFrameTransformer transformer, boolean color) {
```

Comment:
Instead of each caller having to keep track of color selection, it feels like something that the Renderer should hold onto. That's likely a bigger change though.

Reply:
This is an internal method; our public APIs throw `PklException`s with messages that are either colored or not, depending on how they've configured the evaluator